pkg/chezmoi/delims.go

```diff
@@ -0,0 +1,7 @@
+package chezmoi
+
+// Delims are template delimiters.
+type Delims struct {
```

Comment:
As a nit, please rename this to use the full word `Delimiters`, similarly please replace other occurrences of "delims" with "delimiters".